content/en/network_performance_monitoring/network_page.md

```diff
@@ -150,6 +150,36 @@ Select any row from the data table to see associated logs, traces, and processes
 
 {{< img src="network_performance_monitoring/network_page/flow_details.png" alt="Aggregate Connection Details"  style="width:80%;">}}
 
+### Rollups
```

Comment:
This belongs under the sidepanel section, since individual connections are shown in the sidepanel